src/common/util_bug.h

```diff
@@ -40,7 +65,8 @@
 /** Like assert(3), but send assertion failures to the log as well as to
  * stderr. */
 #define tor_assert(expr) STMT_BEGIN                                     \
-  if (PREDICT_UNLIKELY(!(expr))) {                                      \
```

Comment:
Hm, why are we switching from `PREDICT_UNLIKELY` to reverse `ASSERT_PREDICT_LIKELY_` here?

Couldn't we use `ASSERT_PREDICT_UNLIKELY_`? I thought the reason we defined a new one above is that the new one gives us compiler warnings as it should (or that's what its comment implies).

Reply:
If we used `ASSERT_PREDICT_UNLIKELY_()`, we would have to use "!(expr)" instead of expr -- but the extra layer of parentheses would suppress the warning.